### ISSUE TITLE
[Feature] Media gallery: show number of images in total set

### ISSUE THREAD

**Issue by BradHN1** [massenergize/frontend-admin#1100]
**User Story.**
As an admin using media gallery
I want to know how many images I might need to scroll through
So that I don't waste time

**Additional details/context**
The media gallery loads about 32 images per page, but doesn't tell you how many pages there are.

![Screenshot 2023-10-05 at 3 17 42 PM](https://github.com/massenergize/frontend-admin/assets/12800484/5f4074ac-929f-44ae-8355-5c8c297e023c)

The bottom line should show you the number of items out of the total number of items, if possible
